### DISCUSSION TITLE
tuf a15 2023 60hz bug

### DISCUSSION THREAD

**Discussion by omergunek** [seerge/g-helper#2621] [General]
It automatically switches to 60hz mode when not charging. It wasn't like this before, I always want to use 120 Hz. Switching to 60 Hz automatically is a mistake.

**Comment by seerge** [comment]
@omergunek it does not switch refresh rate unless you explicitly selected `Auto`. If it still switches - it's not G-Helper doing that at all.

**Reply by omergunek** [reply]
not automatically selected. I just updated bios 313 yesterday. It wasn't like this before.

**Reply by seerge** [reply]
Then you probably have Asus Smart Display Control running that does that. Please check readme 
https://github.com/seerge/g-helper/wiki/Requirements#recommendations-optional

**Reply by omergunek** [reply]
It really worked, you are very good. Do I need to do anything special to uninstall the Asus Smart Display application?

**Reply by seerge** [reply]
You just need to uninstall it, if you don’t plan to use AC
